index.html

```diff
@@ -1908,10 +1908,6 @@ <h2>Definition of Roles</h2>
 						<th class="role-presentational-inherited-head" scope="row">Inherits Presentational:</th>
 						<td class="role-presentational-inherited"> </td>
 					</tr>
-						<tr>
-							<th class="implicit-values-head" scope="row">Implicit Value for Role:</th>
-							<td class="implicit-values">Default for <sref>aria-checked</sref> is <code class="default">false</code>.</td>
-						</tr>
```

Comment:
I don't think we want to delete these; we have scripts which do the removal when the spec is viewed. See, for example, https://w3c.github.io/aria/#code (shows no implicit values) and https://github.com/w3c/aria/blob/d5d2883/index.html#L2007 (has rows for implicit values).

Reply:
thanks @joanmarie i'll update

Reply:
OK.  should be good to go now.